.pre-commit-config.yaml

```diff
@@ -29,6 +29,8 @@ repos:
         # syntax to notify a line break.
         # See https://github.com/pre-commit/pre-commit-hooks/issues/57
         exclude_types: [markdown]
+        pass_filenames: false
```

Comment:
this hook is only executed on the files that were provided to it, so it makes sense to remove `pass_filenames: false`